### PR TITLE
Add context menu for package cards implement on clicks

### PR DESCRIPTION
**Pull request by jonas-tai**
### Summary of changes
Adds context menu functionality for delete, delete globally, confirm, confirm globally and show resources. Hide is not implemented yet.

### Context and reason for change

Before this commit, the ContextMenu only displayed options, but it was not possible to actually execute any of them. This adds the functionality for delete, delete globally, show resources, confirm and confirm globally, and therefore reduces the amount of necessary clicks by the user to execute actions on attributions. 

### How can the changes be tested

run `yarn test`  and / or try out ContextMenu functionality with example file

### CODE REVIEW

File: src/Frontend/Components/PackagePanelCard/__tests__/PackagePanelCard.test.tsx [hunk 1]
```diff
@@ -166,7 +167,7 @@ describe('The PackagePanelCard', () => {
     fireEvent.click(screen.getByLabelText('show resources'));
     expect(screen.getByText('Resources for selected attribution'));
 
-    fireEvent.click(screen.getByText('Close'));
+    fireEvent.click(screen.getByText(ButtonText.Close));
```
Comment (benedikt-richter) [inline src/Frontend/Components/PackagePanelCard/__tests__/PackagePanelCard.test.tsx:170]: 👍

File: src/Frontend/state/actions/view-actions/__tests__/view-actions.test.ts [hunk 2]
```diff
@@ -127,4 +129,21 @@ describe('popup actions', () => {
     testStore.dispatch(closePopup());
     expect(getOpenPopup(testStore.getState())).toBeFalsy();
   });
```
Comment (benedikt-richter) [inline src/Frontend/state/actions/view-actions/__tests__/view-actions.test.ts:131]: insert empty line

File: src/Frontend/state/actions/view-actions/view-actions.ts [hunk 3]
```diff
@@ -74,3 +79,16 @@ export function setFollowUpFilter(
...
+export function openPopupWithTargetAttributionId(
+  popupType: PopupType,
+  attributionId: string
+): OpenPopupWithTargetAttributionIdAction {
+  return {
+    type: ACTION_OPEN_POPUP_WITH_TARGET_ATTRIBUTION_ID,
+    payload: {
+      popupType: popupType,
+      attributionId: attributionId,
+    },
```
Comment (benedikt-richter) [inline src/Frontend/state/actions/view-actions/view-actions.ts:92]: could be `payload {popupType, attributionId}`

**Review by nicarl — APPROVED** [review]

**Review by benedikt-richter — APPROVED** [review]
some minor fixes will be done on target branch

**Pull request merged**